Servers.yaml

```diff
@@ -0,0 +1,122 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: "Server Infrastructure with Launch Template, Auto Scaling Group, and Application Load Balancer"
+
+Parameters:
+  VpcId:
+    Description: VPC ID for the ALB and servers
+    Type: String
+
+  PublicSubnets:
+    Description: Comma-separated list of public subnet IDs for the ALB
+    Type: List<AWS::EC2::Subnet::Id>
+
+  PrivateSubnets:
+    Description: Comma-separated list of private subnet IDs for the servers
+    Type: List<AWS::EC2::Subnet::Id>
+
+  ALBSecurityGroup:
+    Description: Security Group for the ALB
+    Type: String
+
+  ServerSecurityGroup:
+    Description: Security Group for EC2 Instances
+    Type: String
+
+  KeyName:
+    Description: Key pair name for SSH access
+    Type: AWS::EC2::KeyPair::KeyName
+
+Resources:
+  # Launch Template for EC2 Instances
+  TwitterLaunchTemplate:
```

Comment:
- Storage specs is missing, you need to attach storage using `BlockDeviceMappings`